README.md

```diff
@@ -32,41 +32,96 @@ Configure a webhook for your Github repository following the instructions [here]
 
 Lita has great documentation. Read [this](http://docs.lita.io/plugin-authoring/) first before starting. It will make below instructions easy to understand.
 
-1. Install and start [lita development environment](http://docs.lita.io/getting-started/installation/#development-environment).
-2. Inside the lita development environment:
-  1. Install [ngrok](https://ngrok.com/download).
-  2. Launch ngrok inside the lita dev VM.
-    * `./ngrok http 8080`
-  3. Follow instructions [here](https://dashboard.ngrok.com/get-started) to setup ngrok.
-3. Add a webhook to your github repo per instructions [here](https://developer.github.com/guides/delivering-deployments/).
-  * You can find the ngrok url to use at [ngrok dashboard](https://dashboard.ngrok.com/status).
-4. Copy the github key you would like to use into `/home/lita/.ssh/github` with mod `0600`.
-5. Create `/home/lita/.ssh/config` to use the correct key when talking to Github:
-  ```
-  Host github.com
-  IdentityFile /home/lita/.ssh/github
-  StrictHostKeyChecking no
-  ```
-6. Install git: `apt-get install git`
-7. Put `lita-versioner` into `lita-dev/workspace`.
-8. Get your Jenkins API token by logging in to https://manhattan.ci.chef.co, clicking your name on the top right, and clicking Configure. Then click "Show API Token" and copy the values for the next step.
-9. Create `lita-versioner/lita_config.rb` with this text:
+### First Time Setup
+
+1. Install [Docker Toolbox](https://www.docker.com/products/docker-toolbox):
+   - `brew cask install dockertoolbox`
+   - `docker-machine create default --driver virtualbox`
+   - `eval $(docker-machine env default)` <-- you might want that in your .profile
+2. Copy `lita_config.rb.example` to `lita_config.rb` with this text:
    ```ruby
```

Comment:
It actually works correctly indented, but you have to add a newline between. Fixed everywhere.